sysdump/sysdump.go

```diff
@@ -1170,6 +1171,27 @@ func (c *Collector) submitTetragonBugtoolTasks(ctx context.Context, pods []*core
 	return nil
 }
 
+func (c *Collector) getCiliumVersion(ctx context.Context, p *corev1.Pod) (*semver.Version, error) {
```

Comment:
Thanks @tbalthazar, good suggestion.

FWIW, `ExecInPodWithTTY` will probably go away soon: #1276. I think it'd make sense to use `ExecInPodWithStderr` in both places and move the implementation to a common package.  I'd suggest to move it to the `k8s` package as a `(*Client)` method instead of to the `util` package, though. We already have other helper methods which exec into pods to execute some command, e.g. https://github.com/cilium/cilium-cli/blob/43a7e471eb153942842ab41ee828d03ede7a5c81/k8s/client.go#L410 so this package seems to be a good fit.

Reply:
Hey @tklauser, I extracted the helper and rebased on main. Please let me know what you think. Thanks!